conf/application.conf

```diff
@@ -152,6 +144,7 @@ urls {
   loginHmrcService  = "https://www.gov.uk/log-in-register-hmrc-online-services"
   declareTransitMovementUnloadingFrontend = "http://localhost:9488/common-transit-convention-unloading-arrival"
   declareTransitMovementArrivalFrontend = "http://localhost:9483/common-transit-convention-trader-arrival"
+  declareTransitMovementDepartureFrontend = "http://localhost:9489/common-transit-convention-departure"
```

Comment:
@PSouthern looks like it is not in sync with app-config-base 
declareTransitMovementDepartureFrontend = "/common-transit-convention-departure/local-reference-number"

declareTransitMovementDepartureFrontend is missing /local-reference-number"